app/src/main/java/nl/tudelft/bbw/blockchain/Acquaintance.java

```diff
@@ -15,14 +16,14 @@
     /**
      * Properties of a Acquaintance
      */
-    private ArrayList<Chain> multichain;
+    private List<List<Block>> multichain;
```

Comment:
Initially, there is a class called Chain (extends ArrayList<Block) which if implemented will result in a List<Chain> instead of List<List<Block>> like this. This class was made to serve the purpose of making the code more observable. But the API & BlockControllers were implemented without the Chain class which result in these "ugly" objects. I cannot use Chain anymore because the above reason.